### PR TITLE
Add Synthea postInstall file

### PR DESCRIPTION
**Pull request by josephsnyder**
Add another command line argument which will allow the user to install
a version of the Synthea Ingestor patch using a supplied DUZ value as
the installer.

Add information about the DUZ that needs to be supplied.  **At this point,
it could be added as a command line value for FOIA based instances, where the install will work as user 17. 
 But, it will  not on a VEHU instance due to user 17 not existing.**

### CODE REVIEW

**Comment by josephsnyder** [comment]
Okay!  We've got it down to one post install routine, installing  as user 1, which can be used like all the other postInstalls.  Please give this a try when you are ready.

File: Common/syntheaPostInstall.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+mkdir /tmp/loader
+pushd /tmp/loader
+curl -fsSL --progress-bar https://github.com/OSEHRA/VistA-FHIR-Data-Loader/releases/download/0.1/VISTA_SYNTHETIC_DATA_LOADER_BUNDLE_0P2T1.KID.zip -o loader.zip
```
Comment (ChristopherEdwards) [inline Common/syntheaPostInstall.sh:5]: is there a way to use the github api to always get the latest version? may prevent future updates to this script.

File: Common/syntheaPostInstall.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+unzip -q loader.zip -d /tmp/loader/Synthea
+echo "Installing Synthea ingestor patch"
+system=1
+instanceName='-cn $instance'
+if ($installgtm || $installYottaDB); then
+  system=2
+  instanceName=''
+fi
+if [ ! -d "VistA" ]; then
```
Comment (ChristopherEdwards) [inline Common/syntheaPostInstall.sh]: If i read this right we are in the /tmp/loader directory and looking for the VistA directory, I highly doubt that will ever exist in new installs, is there anywhere else that we download the VistA test repo and check for that instead?

File: Common/syntheaPostInstall.sh [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  echo "Downloading OSEHRA VistA Tester Repo"
+  curl -fsSL --progress-bar https://github.com/OSEHRA/VistA/archive/master.zip -o VistA-master.zip
+  unzip -q VistA-master.zip
+  rm VistA-master.zip
+  mv VistA-master VistA
+fi
+pushd VistA/Scripts/
+# Installs as the DUZ=1 user
+python PatchSequenceApply.py -S $system -p /tmp/loader/Synthea $instanceName -l /tmp/ -i -n ALL -d 1
+popd
```
Comment (ChristopherEdwards) [inline Common/syntheaPostInstall.sh:29]: maybe we should popd before we pushd to VistA/Scripts?

File: Common/syntheaPostInstall.sh (hunk 1 above)
Comment (shabiel) [reply Common/syntheaPostInstall.sh:5]: No. We have to use specific KIDS builds in releases. We are not just getting the source code as there is some pre-install and post-install stuff that needs to be done.

Comment (ChristopherEdwards) [reply Common/syntheaPostInstall.sh:5]: I'm referring to the latest release kids build not source code. Not a big issue, if we lock versions then we should probably have a comment here.

File: Common/syntheaPostInstall.sh (hunk 3 above)
Comment (shabiel) [reply Common/syntheaPostInstall.sh:29]: Looks okay to me; but I will try it.

Comment (ChristopherEdwards) [reply Common/syntheaPostInstall.sh:29]: i'm not saying what is written won't work, just seems messy.

**Comment by shabiel** [comment]
Hmm... I don't feel comfortable doing that right now; plus I do not
anticipate making many new releases (maybe 1 or 2).

**Review by shabiel — APPROVED** [review]
Everything now looks good

**Pull request merged**